cmd/dagger/cloud.go

```diff
@@ -104,6 +115,40 @@ func (cli *CloudCLI) Login(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func createNewOrg(ctx context.Context, cli *cloud.Client, w io.Writer) (*auth.Org, error) {
+	url := "https://dagger.cloud/signup?quickstart=true"
+	err := browser.OpenURL(url)
+	if err != nil {
+		fmt.Fprintf(w, "Unable to open browser automatically, please visit %s to create an organization.\n", url)
+	}
+
+	timer := time.After(15 * time.Second)
```

Comment:
Is 15 seconds long enough to go through the whole quickstart? Shouldn't this be a couple of minutes?

Reply:
this timeout is not for the whole quickstart, but a sensible number to wait until the browser gets opened (or the user opens it if that's not possible) and the organization is created. We call this the "quickstart" flow but the gist of it is that it streamlines account creation basically